### PR TITLE
Dudes

### PR DESCRIPTION
**Pull request by dm-mykhalenych**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,9 +16,10 @@ <h2 class="card-title">Welcome to todo list app</h2>
           <div class="form-group">
             <input type="text" class="form-control" id="text" placeholder="input text">
           </div>
-          <button type="submit" class="btn btn-primary" id="button">Submit</button>
+          <button type="submit" class="btn btn-primary" onclick="myFunction()" id="button">Submit</button>
         </div>
       </div>
+      <div id="tag"></div>
```
Comment (georgelviv) [inline index.html]: Tag has very wide sense. Try always name with some close meaning (for example todo-list, wrapper, etc..) . As this is a wrapper for a list, why not to use <ul>?

File: js/task.js [hunk 2]
```diff
@@ -1,3 +1,8 @@
-var text = document.getElementById('text');
-var button = document.getElementById('button');
-console.log(text, button);
+
+function myFunction() {
+  var text = document.getElementById('text').value;
+  var para = document.createElement("P");
```
Comment (georgelviv) [inline js/task.js]: What means para?

Comment (georgelviv) [reply js/task.js]: Why P is uppercase?

File: js/task.js [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+  var t = document.createTextNode(text);
+  para.appendChild(t);
+  document.getElementById("tag").appendChild(para);
```
Comment (georgelviv) [inline js/task.js]: Each time on submut btn you looking for this tag. Move this logic outside from function

File: index.html (hunk 1 above)
Comment (dm-mykhalenych) [reply index.html]: ok, i fixed this

File: js/task.js (hunk 2 above)
Comment (dm-mykhalenych) [reply js/task.js]: i see

File: js/task.js (hunk 3 above)
Comment (dm-mykhalenych) [reply js/task.js]: if i move var, it doesn't work

File: js/task.js [hunk 4]
```diff
@@ -1,3 +1,8 @@
-var text = document.getElementById('text');
-var button = document.getElementById('button');
-console.log(text, button);
+function myFunction() {
+  var text = document.getElementById('text').value;
+  var paragraf = document.createElement("p");
```
Comment (georgelviv) [inline js/task.js]: dude, `paragraf`  ?) or `paragraph` ?)

File: js/task.js [hunk 5]
```diff
@@ -1,3 +1,8 @@
...
+  var t = document.createTextNode(text);
+  paragraf.appendChild(t);
+  document.getElementById("wrapper").appendChild(paragraf);
```
Comment (georgelviv) [inline js/task.js]: Be consistent, use single quotes, or dobule qoutes.

**Review by georgelviv — APPROVED** [review]

**Pull request merged**